packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1540,7 +1540,11 @@ abstract class RenderBox extends RenderObject {
       return true;
     }());
     _size = value;
-    assert(() { debugAssertDoesMeetConstraints(); return true; }());
+    assert(() {
+      // RenderObject.debugCheckingIntrinsics = false;
+      debugAssertDoesMeetConstraints();
+      return true;
+    }());
```

Comment:
i imagine this is a mistake?

(please consider self-reviewing PRs, especially big ones, so that you can catch issues like this rather than having your reviewer do so.)